dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/eols/NewConditionTransformations.scala

```diff
@@ -0,0 +1,177 @@
+package org.broadinstitute.monster.dap.eols
+
+import org.broadinstitute.monster.dap.common.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.EolsNewCondition
+
+object NewConditionTransformations {
+
+  def mapNewConditionMetadata(rawRecord: RawRecord): EolsNewCondition = {
+    //    val ToxinConsumption = Some(diagnosisType.contains("2"))
+    val diagnosisType = rawRecord.fields.get("eol_dx")
+    val InfectiousDisease = diagnosisType.map(_.contains("1"))
+    val ToxinConsumption = diagnosisType.map(_.contains("2"))
+    val Trauma = diagnosisType.map(_.contains("3"))
```

Comment:
⛏️ All of these var names should be `camelCase`, i.e., 
```suggestion
    val trauma = diagnosisType.map(_.contains("3"))
```